pkg/clusteragent/admission/mutate/autoinstrumentation/auto_instrumentation.go

```diff
@@ -325,6 +326,22 @@ func (w *Webhook) inject(pod *corev1.Pod, _ string, _ dynamic.Interface) error {
 	return w.injectAutoInstruConfig(pod, libsToInject, autoDetected, injectionType)
 }
 
+func injectSecurityClientLibraryConfig(pod *corev1.Pod) {
+	injectEnvVarIfConfigKeySet(pod, "admission_controller.auto_instrumentation.asm.enabled", "DD_APPSEC_ENABLED")
+	injectEnvVarIfConfigKeySet(pod, "admission_controller.auto_instrumentation.iast.enabled", "DD_IAST_ENABLED")
+	injectEnvVarIfConfigKeySet(pod, "admission_controller.auto_instrumentation.sca.enabled", "DD_SCA_ENABLED")
+}
+
+func injectEnvVarIfConfigKeySet(pod *corev1.Pod, configKey string, envVarKey string) {
+	if config.Datadog.Get(configKey) != nil {
```

Comment:
Nit: you canuse `config.Datadog.IsSet(configKey)` instead, which returns `true` if the key exists and `false` otherwise.

Reply:
Yes, good idea. Fixed in e579273691